routes/event_details.js

```diff
@@ -1,17 +1,16 @@
 const request = require('request');
+
 module.exports = (req, res) => {
-  var id = req.params.id;
-  var url = 'https://nazareth-open-tourism-platform.herokuapp.com/events/' + id;
+  const id = req.params.id;
```

Comment:
no need for this to be a separate variable - just put `req.params.id` on the next line, and use a template string plz :v: